app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -269,41 +269,39 @@
   <default>
     <payment>
       <boltpay>
-        <active>1</active>
-        <hide_on_checkout>0</hide_on_checkout>
         <model>boltpay/payment</model>
-        <button_classes>with-cards</button_classes>
-        <title>Credit card</title>
         <order_status>processing</order_status>
         <payment_action>authorize</payment_action>
         <currency>USD</currency>
-        <enable_merchant_scoped_account>0</enable_merchant_scoped_account>
+        <active>1</active>
+        <hide_on_checkout>0</hide_on_checkout>
+        <add_button_everywhere>0</add_button_everywhere>
         <selectors><![CDATA[.btn-proceed-checkout, .minicart-actions .checkout-button, body.firecheckout-index-index .button.btn-checkout]]></selectors>
-        <selector_styles>
-          .btn-proceed-checkout {
-            .bolt-checkout-button.with-cards {
-              float: right;
-            }
-          }
-        </selector_styles>
         <enable_product_page_checkout>0</enable_product_page_checkout>
         <product_page_checkout_selector>.btn-cart</product_page_checkout_selector>
         <additional_css>
-          #checkout-review-submit .bolt-checkout-button {
-            clear:both;
-            float:right;
-          }
-          .col-right.sidebar .block.block-cart .block-content .actions .bolt-checkout-button-button {
-            min-width: max-content!important;
-          }
+#checkout-review-submit .bolt-checkout-button {
+  clear: both;
+  float: right;
+}
+.col-right.sidebar .block.block-cart .block-content .actions .bolt-checkout-button-button {
+  min-width: max-content !important;
+}
         </additional_css>
+        <title>Credit card</title>
+        <button_classes>with-cards</button_classes>
+        <selector_styles>
+.btn-proceed-checkout {
+  .bolt-checkout-button.with-cards {
+    float: right;
+  }
+}
+        </selector_styles>
         <successpage>checkout/onepage/success</successpage>
+        <enable_merchant_scoped_account>0</enable_merchant_scoped_account>
         <allow_po_box>1</allow_po_box>
         <skip_payment>0</skip_payment>
-        <skip_confirm>1</skip_confirm>
         <use_javascript_in_admin>0</use_javascript_in_admin>
-        <add_button_everywhere>0</add_button_everywhere>
-        <sort_order>0</sort_order>
```

Comment:
Where do we render with `sort_order` removed?

Reply:
sort_order is defined in system_xml. I don't know what this is doing here but I'm pretty convinced this doesn't do anything.

Reply:
I'm pretty sure this effects the sort order of the payment methods on the page. But with a value of 0, I also don't think it's doing anything :D